docs/reference/feature-repository/README.md

```diff
@@ -2,6 +2,8 @@
 
 Feast manages two important sets of configuration: feature definitions, and configuration about how to run the feature store. With Feast, this configuration can be written declaratively and stored as code in a central location. This central location is called a feature repository, and it's essentially just a directory that contains some code files.
 
+Feast manages two important sets of configuration: feature definitions, and configuration about how to run the feature store. With Feast, this configuration can be written declaratively and stored as code in a central location. This central location is called a feature repository, and it's essentially just a directory that contains some code files.
```

Comment:
Duplicated ?

Reply:
something messed up with the merge, but this actually came from a separate PR